make/util.cmake

```diff
@@ -16,3 +16,22 @@ function(get_target_property_required var target property)
   endif()
   set(${var} ${PROP} PARENT_SCOPE)
 endfunction()
+
+function(GET_TEMPLATE_FILENAME var SRC PREFIX)
```

Comment:
Please add comment describing this, also this problem belongs close to the ntemplate function

Reply:
I added a description and moved the function to gen.cmake